mod_wasm/modules/wasm/mod_wasm.c

```diff
@@ -478,6 +509,13 @@ static const char *wasm_directive_WasmEnableCGI(cmd_parms *cmd, void *mconfig, i
     return NULL;
 }
 
+static const char *wasm_directive_WasmMapCGIFileNames(cmd_parms *cmd, void *mconfig, int arg)
+{
+    x_cfg *cfg = (x_cfg *) mconfig;
+    cfg->bWasmMapCGIFileNames = arg;
```

Comment:
@juanjoib true, it will be annoying, but I agree with @gzurl on this one:
 - having some active directives, which do nothing is an unnecessary cognitive load on the one who reads the configuration - if you don't know that this directive is ignored, your brain has to read those directives, then start thinking "What is this doing if CGI is not enabled?", then you go read the docs only to find out it's doing nothing, or even if this is not documented you start experimenting by commenting/uncommenting to see if it changes a thing....
 - you wouldn't expect one to change httpd.conf all the time, so the inconvenience of toggling all CGI related directives together won't be a daily thing for the end user, but only when debugging something specific (related to CGI mode vs non-CGI mode) during active development. What I mean is that the annoying inconvenience would happen rarely.

Reply:
Leaves as-is after team discussion, we will add explicit information in the documentation